### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by priiduonu**
Make it clear that brands for custom components are not accepted.

## Type of change

- [ ] Add a new logo or icon for a new integration
- [ ] Add a missing icon or logo for an existing integration
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: 
- Link to documentation pull request: 
- Link to integration documentation on our website: 

## Checklist

- [ ] The added/replaced image(s) are **PNG**
- [ ] Icon image size is 256x256px (`icon.png`)
- [ ] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,8 @@
 This repository holds the icons and logos for all the brands Home Assistant
 supports.
 
+Please do not submit icons and logos for custom components.
```
Comment (frenck) [inline README.md:8]: I don't think this should be in the introduction text of the repository.

Down below it states:

> The `./src` folder contains a folder for each `domain` Home Assistant provides
an integration for.

Home Assistant does not provide custom integrations. However, we could extend that sentence maybe with:

> The `./src` folder contains a folder for each `domain` Home Assistant provides
an integration for. Custom integration domains are not provided by Home Assistant and thus not in this repository.

Comment (priiduonu) [reply README.md:8]: I think some further explanation would help as it seems there are many PR-s related to custom components. The current introduction states what the repo contains but it does not explicitly state what it does **not** contain.

Comment (frenck) [reply README.md:8]: It states that already: "icons and logos for all the brands Home Assistant supports."

Custom integration are not supported by Home Assistant.

I'm fine with adding an additional note, but not in the introduction text of the repository (as said before).

Comment (priiduonu) [reply README.md:8]: Your version of the extended sentence is totally fine, I did not mean to resist it but instead explain the need of some additional information. Feel free to add your version and close this PR.

**Comment by frenck** [comment]
Closing this PR as we are preparing to allow for custom components.

**Pull request closed without merging**